code/iaas/config-item/server/src/main/java/io/cattle/platform/configitem/context/data/metadata/common/ServiceMetaData.java

```diff
@@ -35,6 +35,7 @@
     protected Integer scale;
     protected String fqdn;
     protected List<String> expose = new ArrayList<>();
+    protected String dns_token;
```

Comment:
Yes was using camel case, but in the answers.yml it came up in camel case instead of _ separated. So changed it to using _ just as other fields here.

But anyway, moot